src/agent.js

```diff
@@ -1,6 +1,15 @@
+import User from './user';
+
 class Agent extends User {
-  constructor(agentType) {
-    super(agentType);
+  constructor(agentType, allTrips) {
+    super(agentType, allTrips);
+    this.name = 'Agent Michael Scarn';
+    this.totalRevenue = this.getTotalAmountSpentThisYear(allTrips);
+  }
```

Comment:
I saw you hardcoded the name of Michael Scarn as an Agent name.  I think you can make it more dynamic if you pass in an argument to assign an agent name.  There can be multiple agents that can use this application, so it would make sense to allow any agent to use this application and setting their name into the Agent class.  Just a minor change for overall user's usability.

Reply:
That is a good suggestion, I think i'll take your advise and create a variable name to make it more dynamic. Thanks!